core/page-assets/loader.php

```diff
@@ -35,6 +35,11 @@ private function init_assets() {
 					'version' => ELEMENTOR_VERSION,
 					'dependencies' => [],
 				],
+				'e-apple-webkit' => [
+					'src' => $this->get_css_assets_url( 'appleWebkit', 'assets/css/', true ),
```

Comment:
```suggestion
					'src' => $this->get_css_assets_url( 'apple-webkit', 'assets/css/', true ),
```
Maybe? Just for consistency.

Reply:
Updated.